### PR TITLE
feat: add scrollX, scrollY option

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/view/bodyArea.tsx [hunk 1]
```diff
@@ -166,7 +172,7 @@ class BodyAreaComp extends Component<Props> {
 export const BodyArea = connect<StoreProps, OwnProps>((store, { side }) => {
   const { column, rowCoords, dimension, viewport } = store;
   const { totalRowHeight } = rowCoords;
-  const { bodyHeight, scrollXHeight } = dimension;
+  const { bodyHeight, scrollXHeight, scrollY, scrollX } = dimension;
```
Comment (seonim-ryu) [inline src/view/bodyArea.tsx]: 왠지 `scrollX , scrollY` 이 순서대로 가져오고 싶네요ㅋㅋ

File: src/view/container.tsx [hunk 2]
```diff
@@ -194,11 +196,13 @@ export class ContainerComp extends Component<Props> {
...
-      !!summaryHeight,
-      summaryPosition === 'top' ? 'has-summary-top' : 'has-summary-bottom'
-    ]);
+    const { summaryHeight, summaryPosition, scrollY, scrollX } = this.props;
+    return cls(
+      'content-area',
+      [!!summaryHeight, summaryPosition === 'top' ? 'has-summary-top' : 'has-summary-bottom'],
+      [!scrollX, 'no-scroll-x'],
+      [!scrollY, 'no-scroll-y']
+    );
```
Comment (seonim-ryu) [inline src/view/container.tsx]: 근데 이 메서드는 클래스명만 반환하고 호출하는 곳도 한군데 뿐이라..
`render` 함수에 있어도 좋을 것 같습니다.

**Comment by seonim-ryu** [comment]
[5/31] 리뷰 완료! 고생하셨습니다 👍

**Comment by js87zz** [comment]
리뷰 완료입니다!고생하셨습니다.

File: src/view/bodyArea.tsx (hunk 1 above)
Comment (jung-han) [reply src/view/bodyArea.tsx]: 오 수정하겠습니다ㅋㅋㅋㅋㅋ

File: src/view/container.tsx (hunk 2 above)
Comment (jung-han) [reply src/view/container.tsx]: 넵 render함수로 옮기도록 하겠습니다~

**Pull request merged**